workspace/api/openfdd_bridge/playground_worker.py

```
@@ -0,0 +1,80 @@
+    raise ValueError(f"unknown playground job op: {op!r}")
+
+
+def main(argv: list[str] | None = None) -> int:
+    args = argv if argv is not None else sys.argv[1:]
+    if len(args) != 2:
+        print("usage: python -m openfdd_bridge.playground_worker <job.pkl> <result.pkl>", file=sys.stderr)
+        return 2
+    job_path = Path(args[0])
+    result_path = Path(args[1])
+    try:
+        job = pickle.loads(job_path.read_bytes())
```

Ruff (0.15.15)

[error] `pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)